packages/cli/src/commonlib/cacheAccess.ts

```diff
@@ -145,15 +145,19 @@ export class CryptoCachePlugin {
           }
         }
       } catch (err) {
-        VsCodeLogInstance.error("read token fail: " + err.message);
+        VsCodeLogInstance.warning(
+          "Cannot read token from cache, clean up account cache. " + err.message
```

Comment:
`clean up account cache` is a user action? Does user need to do something for this warning?

Reply:
No, it's acted by toolkit. No user action needed, so make this warning.